conformance/tests/Dockerfile

```diff
@@ -1,4 +1,6 @@
 # syntax=docker/dockerfile:1.5
+# this is here so we can grab the latest version of kind and have dependabot keep it up to date
+FROM kindest/node:v1.27.3
```

Comment:
Does the most recently seen FROM statement get used (golang:1.20) when the image is built, so this statement is ignored by the builder?